<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>Find </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Select </string>
   </property>
   <widget class="QRadioButton" name="rdoName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoQuantity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoSellingPrice">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Selling Price</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoId">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtValue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
